### PR TITLE
[Docs] fix on_after_backward example

### LINKED ISSUE #5238: How to get average of absolute gradient during the training
## ❓ Questions and Help
I would like to calculate the average of absolute value of gradients for a neural network during the training so that I make sure gradients do not vanish or explode during the training. What would be the way to do it? 

In Pytorch,  one can do something like below to collect the gradient after `backward()` is called:

```
optimizer.zero_grad()

pred = model(input)
loss = loss_fn(pred, y)

loss.backward()
grads = []
for param in model.parameters():
    grads.append(param.grad.view(-1))

grads = torch.cat(grads).abs().mean()

```

Since PyTorch Lightning takes care of `backward` behind the scene, I really appreciate it if someone could help. Thanks.

### PR DESCRIPTION
**Pull request by rohitgr7**
## What does this PR do?

Fixes #5238

## Before submitting
- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes [if needed]?
- [ ] Did you write any new necessary tests [no need for typos, docs]? 
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:  

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified
 - [x] **Check that target branch and milestone are aligned!**
 

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by ydcjeff — APPROVED** [review]

**Review by Borda — APPROVED** [review]

**Comment by Borda** [comment]
@rohitgr7 it seems it does only formating, do not touches any docs...

**Comment by rohitgr7** [comment]
@Borda it was `grads = v` before which is wrong, changed it to `grads = v.grad`.

**Review by SkafteNicki — APPROVED** [review]

**Review by justusschock — APPROVED** [review]

**Review by tchaton — APPROVED** [review]

**Comment by rohitgr7** [comment]
@williamFalcon mind review

**Review by williamFalcon — APPROVED** [review]
